dds_web/web/user.py

```diff
@@ -6,15 +6,18 @@
 
 # Standard Library
 import io
+import json
 
 # Installed
 import flask
+from dds_web.api.db_connector import DBConnector
 import flask_login
 import pyqrcode
 import pyotp
 import itsdangerous
 import sqlalchemy
 import marshmallow
+import logging
```

Comment:
This import is not needed I think?

Reply:
According to VS Code, also `flask_mail` and `pyotp` are not used. Should I delete them as well?